### DISCUSSION TITLE
Native postgres enums

### DISCUSSION THREAD

**Discussion by alec-c4** [jeremyevans/rodauth#310] [General]
Hey!
Is it possible to use [native PostgreSQL enums](https://blog.saeloun.com/2022/01/12/rails-7-adds-custom-enum-support-in-postgresql) for account status?

**Comment by jeremyevans** [comment]
Probably, but I haven't tested it.  You might need to use `account_open_status_value`/`account_unverified_status_value`/`account_closed_status_value` to set the expected status values appropriately.
